src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
```

Comment:
Only one instance of class Lottery should exist, which will generate new balls each time you call the method getRandomBall()

Reply:
Use for loop for calling getRandomBall() and printing the result.
Also, calling .toString() is redundant.